src/topology/algebra/field.lean

```diff
@@ -122,6 +123,37 @@ lemma continuous_units_inv : continuous (λ x : Kˣ, (↑(x⁻¹) : K)) :=
 
 end topological_division_ring
 
+section subfield
+
+variables {α : Type*} [field α] [topological_space α] [topological_division_ring α]
+
+/-- The (topological-space) closure of a subfield of a topological field is
+itself a subfield. -/
+def subfield.topological_closure (K : subfield α) : subfield α :=
+{ carrier := closure (K : set α),
+  inv_mem' :=
+  begin
+    intros x hx,
+    by_cases h : x = 0,
+    { rwa [h, inv_zero, ← h], },
+    { convert mem_closure_image (continuous_at_inv₀ h) hx using 2,
+      ext x, split,
+      { exact λ hx, ⟨x⁻¹, ⟨K.inv_mem hx, inv_inv x⟩⟩, },
+      { rintros ⟨y, ⟨hy, rfl⟩⟩, exact K.inv_mem hy, }},
+  end,
+  ..K.to_subring.topological_closure, }
+
+lemma subfield.subfield_topological_closure (s : subfield α) :
```

Comment:
```suggestion
protected lemma subfield.le_topological_closure (s : subfield α) :
```

Reply:
Well, for this lemma, I copied the existing version for `subring`: https://leanprover-community.github.io/mathlib_docs/topology/algebra/ring.html#subring.subring_topological_closure

I don't mind changing the name of the lemma but in this case, it makes sense to change it also for `subring` and there is also a version for `subsemiring`.

Reply:
eww, gross, you're right. I feel like they are *all* misnamed. I'll let another reviewer comment. My feeling would be, if another reviewer agrees, to name this correctly here with `le` and then in another PR change the names of everything else.

Reply:
Since (at least) two reviewers agree, I decided that it was the right thing to do and renamed the lemma. I'll prepare another PR with name changes for all the similar results for other structures.